bin/metrics.js

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+var fs = require('fs')
+var section
+
+module.exports = {
+  run: run,
+  countAccounts: countAccounts,
+  countVerifiedAccounts: countVerifiedAccounts,
+  countAccountsWithTwoOrMoreDevices: countAccountsWithTwoOrMoreDevices,
+  countAccountsWithThreeOrMoreDevices: countAccountsWithThreeOrMoreDevices,
+  countAccountsWithMobileDevice: countAccountsWithMobileDevice
+}
+
+if (require.main === module) {
+  module.exports.run(
+    fs.readFileSync(
+      process.argv[2] || '/etc/gather_basic_metrics.conf',
+      { encoding: 'utf8' }
+    )
+    .split('\n')
+    .map(trim)
+    .filter(filterConfig)
+    .reduce(reduceConfig, {})
+  )
+}
+
+function run (config, now) {
+  now = now || new Date()
+  var lastMidnight = Date.UTC(
+    now.getUTCFullYear(),
+    now.getUTCMonth(),
+    now.getUTCDate(),
+    0, 0, 0, 0
+  )
+  var os = require('os')
+  var log = require('../lib/logging')('bin.metrics')
+  var mysql = require('../lib/db/mysql')(log, require('../fxa-auth-db-server').errors)
+  var P = require('../lib/promise')
+  var self = this
+
+  return mysql.connect({
+    master: {
+      host: config.General.db_dnsname,
+      user: config.General.db_username,
+      password: config.General.db_password,
+      database: config.General.db_name
+    },
+    slave: {
+      host: config.General.db_dnsname,
+      user: config.General.db_username,
+      password: config.General.db_password,
+      database: config.General.db_name
+    },
+    patchKey: 'schema-patch-level'
+  })
+  .then(function (db) {
+    return P.all([
+      self.countAccounts(db, lastMidnight),
+      self.countVerifiedAccounts(db, lastMidnight),
+      self.countAccountsWithTwoOrMoreDevices(db, lastMidnight),
+      self.countAccountsWithThreeOrMoreDevices(db, lastMidnight),
+      self.countAccountsWithMobileDevice(db, lastMidnight)
+    ])
+  })
+  .then(function (counts) {
+    fs.appendFileSync('/media/ephemeral0/fxa-admin/basic_metrics.log', JSON.stringify({
+      hostname: os.hostname(),
+      pid: process.pid,
+      op: 'account_totals',
+      total_accounts: counts[0].count,
+      total_verified_accounts: counts[1].count,
+      total_accounts_with_two_or_more_devices: counts[2].count,
+      total_accounts_with_three_or_more_devices: counts[3].count,
+      total_accounts_with_mobile_device: counts[4].count,
+      time: (new Date(lastMidnight)).toISOString(),
+      v: 0
+    }))
+  })
+  .catch(function (error) {
+    log.error({ op: 'metrics.run', err: error })
+  })
+}
+
+function countAccounts (db, createdBefore) {
+  return metricsQuery(db, [
```

Comment:
In lieu of ES6 template strings, I opted for `Array.join` as the most readable approach to embedding multi-line strings. Happy to do it differently if you think this way mings though.